requirements.txt

```diff
@@ -27,3 +27,4 @@ rlp>=1.0.1
 transaction>=2.2.1
 z3-solver-mythril>=4.8.4.1
 pysha3
+matplotlib
```

Comment:
Are we going to draw graphs from mythril? Also this should be included in setup.py right?

Reply:
Only developers of mythril will probably have intrest in this feature, there is also not really a way to enable this module atm without changing the code.

But we could add it to the setup.py as well